### ISSUE TITLE
Document training hyper-parameters and recommended settings

### ISSUE THREAD

**Issue by thesfy** [ICSResearch/SSUN-Net#17]
Important hyper-parameters (learning rate, batch size, number of epochs, regularization / optimizer settings) are not clearly documented for different datasets or configurations.
